app/controllers/hooks_controller.rb

```diff
@@ -118,6 +118,29 @@ def temps
              status: 405
     end
   end
+  def heat
+    if heat_params['event'].present? && heat_params['event'].start_with?('heat')
+
+      @heat_status = HeatingElementStatus.new heat_params
+      if @heat_status.save
+        render json: {
+                   msg: 'Heating Element status logged!'
+               },
+               status: 200
+      else
+        render json: {
+                   msg: 'Heating Element status could not be logged for some reason...'
+               },
+               status: 500
+      end
+    else
+      render json: {
+                 msg: 'Specified event cannot be processed at this endpoint.',
+                 name: heat_params[:name]
+             },
+             status: 405
+    end
+  end
 
   private
 
```

Comment:
You need to provide the heat_params method that you're referencing. It should look similar to pump_params and temp_params below.